### PR TITLE
Added flow for adding osd to existing osd node

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by r0h4n** [comment]
I was expecting an OSD object and a Journal object, is this PR incomplete?

**Comment by shtripat** [comment]
>> I was expecting an OSD object and a Journal object, is this PR incomplete?

Latest updated PR has journal object defined and for OSD object syncing I have a PR at https://github.com/Tendrl/ceph-integration/pull/227

**Comment by shtripat** [comment]
With latest ceph-installer today and enough OSDs in the cluster (at least 2 in the starting), I am able to add OSDs to the existing OSD node. Works as expected.

File: tendrl/integrations/ceph/flows/add_osds/__init__.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+from tendrl.commons.flows.exceptions import FlowExecutionFailedError
+from tendrl.commons.flows.create_cluster import \
+    ceph_help as create_ceph_help
+from tendrl.commons.flows.expand_cluster import ceph_help
+from tendrl.commons.message import Message
+
+
+class AddOsds(flows.BaseFlow):
+    def run(self):
+        import pdb; pdb.set_trace()
```
Comment (r0h4n) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: remove PDB

Comment (shtripat) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: done

File: tendrl/integrations/ceph/flows/add_osds/__init__.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    # If osds passed create and add them
+    if len(osd_ips) > 0:
+        Event(
+            Message(
+                job_id=parameters['job_id'],
+                flow_id=parameters['flow_id'],
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Creating Ceph OSD %s" %\
```
Comment (r0h4n) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: Add the osd_ips in log msg

File: tendrl/integrations/ceph/flows/add_osds/__init__.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        )
+        create_ceph_help.create_osds(parameters, created_mons)
+        Event(
+            Message(
+                job_id=parameters['job_id'],
+                flow_id=parameters['flow_id'],
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Created OSD on Cluster %s" %\
```
Comment (r0h4n) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: same as above ^

File: tendrl/integrations/ceph/flows/add_osds/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: done

File: tendrl/integrations/ceph/flows/add_osds/__init__.py (hunk 3 above)
Comment (shtripat) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: done

**Pull request merged**